inspirehep/modules/theme/jinja2filters.py

```diff
@@ -590,6 +590,13 @@ def is_external_link(url):
             ('.pdf', '.png', '.jpg', '.jpeg')))
 
 
+@blueprint.app_template_filter()
+def is_institute(institute):
```

Comment:
the name is wrong, we don't call this an institute any more in the schema. Maybe `is_known_exernal_system_identifiers`? but the way this whole thing is done is wrong IMHO.
Why don't you pull the whole logic to link to external system identifiers in the record wrapper, and have there a property which is a `dict` `link text: link url` and only exposes linkable `external_system_identifiers`?
Right now, if you want to add a new one (e.g. `HepData` is missing), you need to amend both this function and the template.

Reply:
@jmartinm @michamos I've made some changes to this PR as per your suggestions. Most of the logic is now in wrappers.py